### PR TITLE
add funder page / logos

### LINKED ISSUE #97: Add Funder page / logos

### PR DESCRIPTION
**Pull request by wenxich**
closes #97

desktop view:
Screenshot 2024-06-15 at 12 19 40 AM
mobile view:
Screenshot 2024-06-15 at 12 15 46 AM

a couple of questions:
- should movement alliance project (fiscal sponsor) go on this page too?
- the images are links pulled from online sources, but if we’d prefer to have a local copy, i can add a funders folder to static/images - not entirely necessary but in case we want to play it safe
- let me know if there’s anything i should improve aesthetically.

### CODE REVIEW

**Comment by hawc2** [comment]
thanks @wenxich, this looks great! I agree storing the images locally would be better, let's do that. We can hold off on adding MAP for now. 

The main thing that would be helpful is if you can add a description to the readme of how to add the logos this way, so @leannepcw can work on additional edits as we clarify where things will go on the website. I agree MAP should be listed somewhere with their logo, but it could be in a different section than 'funders'

**Comment by hawc2** [comment]
@eugenethreat I added you as a reviewer so onc @wenxich makes these changes, could you preview the site locally and make sure everything works before merging the pull request?

**Review by eugenethreat — APPROVED** [review]
All looks good to me! The only issue I can see is in the future, as we add more sections the navbar is going to get bigger and bigger - opening a new issue for adjusting the UI on mobile ... 
image

**Pull request merged**
